docs/shared/getting-started/intro.md

```diff
@@ -1,118 +1,85 @@
+{% youtube
+src="https://www.youtube.com/embed/-_4WMl-Fn0w"
+title="What is Nx?"
+width="100%" /%}
 
-<!-- - **Monorepo and Single Projects**: Nx supports both, monorepos as well as single-project (standalone) workspaces. -->
+## How Nx solves scaling
 
-Find out more about [why you should use Nx](/getting-started/why-nx) or browse our [features](/features).
+Monorepos offer significant advantages. They reduce integration overhead between different software components, enabling faster cross-project changes, and improving code sharing and collaboration among teams. However, without proper tooling, as projects multiply, build times increase and management complexity grows, potentially negating the original benefits.
 
-## Try Nx Yourself!
+Nx solves this by **providing solutions along the way for each step**, whether it's [for local development](/features), [integrating with your editor](/getting-started/editor-setup), [making your AI assistant smarter](/getting-started/ai-integration) and [scaling CI runs](/ci/features).
 
-{% side-by-side %}
+## Start small, extend as you grow
 
-```shell {% title="Create a new workspace" %}
-npx create-nx-workspace
-```
-
-```shell {% title="Add Nx to an existing workspace" %}
-npx nx init
-```
-
-{% /side-by-side %}
-
-Also, here are some recipes that give you more details based on the technology stack you're using:
-
-{% cards cols="2" mdCols="4" smCols="2" moreLink="/recipes/adopting-nx" %}
-
-{% link-card title="Add to Existing Monorepo" appearance="small" url="/recipes/adopting-nx/adding-to-monorepo" icon="pnpm" /%}
-
-{% link-card title="Add to Any Project" appearance="small" url="/recipes/adopting-nx/adding-to-existing-project" icon="nx" /%}
+Nx is built in a modular fashion, allowing you to adopt as little or as much as you'd like at any moment in your development lifecycle. You can **start with just the core and add additional capabilities incrementally** as your needs grow and complexity increases.
 
-{% link-card title="Migrate from Angular CLI" appearance="small" url="technologies/angular/migration/angular" icon="angular" /%}
+**At the foundation is Nx Core**: a Rust-based, technology-agnostic task runner that builds a knowledge graph of your workspace, understanding project relationships and dependencies. This enables highly optimized and fast task execution regardless of technology stack. It runs `package.json` scripts in [TypeScript monorepos](/technologies/typescript/introduction) or Gradle tasks in [Java projects](/technologies/java/introduction).
 
-{% link-card title="Import Projects into Your Nx Workspace" appearance="small" url="/recipes/adopting-nx/import-project" icon="ArrowDownOnSquareStackIcon" /%}
+{% callout type="deepdive" title="Running NPM Scripts" %}
```

Comment:
I'm really split on calling out the deep dive in the intro page. I think it's fine for now. but overall I want to make things as simple and straight forward on these initial pages and this feels almost like a diversion into people trying to grasp Nx off the bat.